src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -109,6 +109,12 @@ public sealed class Driver
         private static readonly OptionInfo<string> JobNameOption = new OptionInfo<string>(
             ImmutableList.Create("--job-name"), Maybe.Just(string.Empty), "The name of the submitted job.");
 
+        /// <summary>
+        /// The target capability option.
+        /// </summary>
+        private static readonly OptionInfo<string> TargetCapabilityOption = new OptionInfo<string>(
+            ImmutableList.Create("--target-capability"), Maybe.Just(string.Empty), "The name of the target capability.");
```

Comment:
Come to think of it, I don't think it makes sense to add this to the command line options for the executable; The Q# compiler will (and should) populate what is currently called DefaultTargetCapability in the DriverSettings based on the project information. We will want to pass that information to the AzureSettings. But a user cannot really change what a program has been compiled for after it was compiled, so adding it to the command line for the executable doesn't make sense since it is already built for a specific TargetCapability.

Reply:
Changed.